applications/logan_progress_report/logan_progress_report.org

```diff
@@ -0,0 +1,275 @@
+#+LATEX_HEADER: \input{styles.tex}
+#+TITLE: hackEDU Progress Report
+* Accomplishments to Date
+** Hack Camp
+We took this summer to run Hack Camp, a two-week summer program that brought
+local high school students from little or no coding experience to publishing
+their first websites and games. In running Hack Camp we had two goals:
+
+1. Find a model for running Hack Camp
+2. Have high school students reproduce it
+
+This was part of our broader goal of using Hack Camp to build the first
+iteration of our club playbook.
+
+We ran four sessions of Hack Camp, ranging in size from 3 to 12 hackers. We
+spent the first three sessions iterating on curriculum until we found a model
+that we were confident in.
+
+For the final session, we built set of materials for running Hack Camp (called
+the "Hack Camp playbook") and had previous attendees of Hack Camp, along with
+one of our high school volunteers from LA, run it from that playbook
+successfully.
```

Comment:
This sentence (19-22) seems rather long. Perhaps shortening it or making it more concise would help?